lib/index.js

```diff
@@ -171,10 +171,13 @@ bedrock.events.on('bedrock.start', async () => {
   // ready check
   const cancel = await bedrock.events.emit('bedrock-express.ready', server);
   // check if default attach canceled
-  if(cancel !== false) {
+  if(!bedrock.config.express.httpOnly || cancel !== false) {
```

Comment:
I'm confused by this -- which may be related to the original code as well. Is there some kind of double negative going on here? Previously, if `cancel` was anything but `false`, we would attach express to the HTTPS server. I think that was probably backwards given the variable name. My understanding is that a return value of `false` means the event *was* canceled. So perhaps we should do:

```js
const canceled = await bedrock.events.emit('bedrock-express.ready', server) === false;
```

 And then:

```suggestion
  if(!bedrock.config.express.httpOnly && !canceled) {
```

Which would then read more clearly as: If `httpOnly` isn't set and we haven't canceled, attach to HTTPS.